index.js

```diff
@@ -1,68 +1,79 @@
 module.exports = {
     env: {
-        'jasmine': true
+        es6: true,
+        jasmine: true,
+        node: true
     },
-
-    // set this as the base `.eslint` file
-    root: true,
-
     // start with a sane baseline
     extends: 'eslint:recommended',
-
+    parserOptions: {
+        ecmaVersion: 9,
+        sourceType: 'module',
+    },
     plugins: ['jasmine'],
-
-    // linting rules settings
+    // set this as the base `.eslint` file
+    root: true,
     rules: {
         'array-bracket-spacing': ['error', 'never'],
         'block-spacing': ['error', 'always'],
         'brace-style': ['error', '1tbs', {
-            'allowSingleLine': true
+            allowSingleLine: true
         }],
-        'camelcase': ['error', { 'properties': 'never' }],
+        camelcase: ['error', { properties: 'never' }],
         'comma-spacing': ['error', {
-            'before': false, 'after': true
+            before: false, after: true
         }],
         'comma-style': ['error', 'last'],
+        curly: ['error', 'all'],
```

Comment:
Only one I'm not sure about is this one. If its abused yes it can be messy but sometimes one liners are nice.

Reply:
personally I love it

Reply:
they are error prone, and require more effort to refactor

Reply:
👍